SPECS/kexec-tools/kexec-tools.spec

```diff
@@ -188,13 +185,12 @@ install -m 644 %{SOURCE16} $RPM_BUILD_ROOT%{_unitdir}/kdump.service
 install -m 755 -D %{SOURCE22} $RPM_BUILD_ROOT%{_prefix}/lib/systemd/system-generators/kdump-dep-generator.sh
 
 %ifarch %{ix86} x86_64 aarch64
-install -m 755 makedumpfile-%{mkdf_ver}/makedumpfile $RPM_BUILD_ROOT/usr/sbin/makedumpfile
-install -m 644 makedumpfile-%{mkdf_ver}/makedumpfile.8.gz $RPM_BUILD_ROOT/%{_mandir}/man8/makedumpfile.8.gz
-install -m 644 makedumpfile-%{mkdf_ver}/makedumpfile.conf.5.gz $RPM_BUILD_ROOT/%{_mandir}/man5/makedumpfile.conf.5.gz
-install -m 644 makedumpfile-%{mkdf_ver}/makedumpfile.conf $RPM_BUILD_ROOT/%{_sysconfdir}/makedumpfile.conf.sample
+mkdir -p $RPM_BUILD_ROOT/usr/sbin
```

Comment:
```suggestion
mkdir -p %{buildroot}/usr/sbin
```

Instead of `$RPM_BUILD_ROOT/`, use `%{buildroot}` which is the AzLinux macro. If it is helpful, you can check for RPM macros in a running AzLinux image by using `rpm --eval`

<img width="269" alt="image" src="https://github.com/microsoft/CBL-Mariner/assets/10325590/ce89faf7-6094-4af2-b9f1-8053e0b6ec37">

Reply:
This spec has used this RPM_BUILD_ROOT in first place and I preferred to use this use this PR for it's purpose which is "upgrading the makedumpfile tool".